pom.xml

```diff
@@ -99,6 +87,19 @@
     </profile>
   </profiles>
 
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
```

Comment:
What is `rh`?

Reply:
`OSS Repository Hosting` - I copied it from Sonatype docs basically 😅